subsys/bluetooth/controller/Kconfig

```
@@ -202,12 +202,12 @@ config BT_CTLR_ISO_TX_BUFFERS
 config BT_CTLR_ISO_TX_BUFFER_SIZE
 	int "Isochronous Tx buffer size"
 	depends on BT_CTLR_ADV_ISO || BT_CTLR_CONN_ISO
-	range 1 4095
-	default BT_ISO_TX_MTU if BT_ISO
+	range 1 65535
```

```suggestion
	range 1 16383
```

While the HCI command states that it has a range of 0x01 - 0xFFFF the `Data_Total_Length` is only 14 bits, so it doesn't really make sense to return/allocate buffers larger than that. 

In fact, since the ISO_SDU_Length is only 12 bits, the actual maximum buffer size is 4095 + 8 (timestamp, psn, length). 

We can keep it as 0xFFFF in case that the RFU bits will be used for the packet length in the future.

---

@cvinayak Will you follow through with this change? Or should it be 4095 + 8 or 16383?

---

@Thalley i make the range be 9 to 4103 for `BT_LL_SW_SPLIT`  and left the HCI Spec range otherwise.